accessibility-checker-engine/help/combobox_focusable_elements.mdx

```diff
@@ -8,11 +8,11 @@ import { CodeSnippet, Tag } from "carbon-components-react";
 <Row>
 <Column colLg={16} colMd={8} colSm={4} className="toolHead">
 
-### Tabbable focus is not set correctly within the combobox or related popup
+### Tabable focus is not set correctly within the combobox or related popup
```

Comment:
Tabbable was correct

Reply:
agree. This one shoud not have been changed. correcting back